assets/css/layout/nav.less

```diff
@@ -1,8 +1,7 @@
 .nav {
   display: none;
   width: 100%;
-  height: 70px;
-  padding: 1rem 0;
+  padding: 0.3rem 0;
```

Comment:
When the hero's visible, the spacing on the home page nav doesn't look right. Can we bring it back to its original state when the hero's visible so it's balanced?

<img width="641" alt="screen shot 2017-05-03 at 6 49 07 pm" src="https://cloud.githubusercontent.com/assets/7584833/25686091/6403ecac-3031-11e7-82a2-95bb341c4833.png">